cg/apps/scout/scoutapi.py

```diff
@@ -184,67 +172,51 @@ def get_cases(
-            cases (list): list of cases
+        Get cases solved within chosen time span.
         """
         return self.get_cases(status="solved", days_ago=days_ago)
 
     def upload_delivery_report(self, report_path: str, case_id: str, update: bool = False) -> None:
-        """Load a delivery report into a case in the database
-
+        """Load a delivery report into a case in the database.
         If the report already exists the function will exit.
         If the user want to load a report that is already in the database
-        'update' has to be 'True'
-
-        Args:
-            report_path (string):       Path to delivery report
-            case_id     (string):       Case identifier
-            update      (bool):         If an existing report should be replaced
-
-        Returns:
-            updated_case(dict)
-
+        'update' has to be 'True'.
         """
-        # This command can be run with `scout load delivery-report <CASE-ID> <REPORT-PATH>`
-        upload_command = ["load", "delivery-report", case_id, report_path]
+
+        upload_command: List[str] = ["load", "delivery-report", case_id, report_path]
 
         if update:
             upload_command.append("--update")
 
         try:
-            LOG.info("Uploading delivery report %s to case %s", report_path, case_id)
+            LOG.info(f"Uploading delivery report {report_path} to case {case_id}")
             self.process.run_command(upload_command)
         except CalledProcessError:
             LOG.warning("Something went wrong when uploading delivery report")
 
+    def upload_report(self, case_id: str, report_path: str, report_type: str) -> None:
+        """Load report into a case in the database. If the report already exists the function will exit."""
+
+        upload_command: List[str] = ["load", "report", "-t", report_type, case_id, report_path]
+
+        try:
+            LOG.info(f"Uploading {report_type} report to case {case_id}")
+            self.process.run_command(upload_command)
+        except CalledProcessError:
+            LOG.warning("Something went wrong when uploading multiqc report")
+
     def upload_fusion_report(
         self, case_id: str, report_path: str, research: bool, update: bool
     ) -> None:
-        """Load a fusion report into a case in the database
-
-        Args:
-            report_path (string):       Path to delivery report
-            case_id     (string):       Case identifier
-            research    (bool):         Research report
-            update      (bool):         If an existing report should be replaced
-        Returns:
-            Nothing
-        """
+        """Load a fusion report into a case in the database."""
 
-        # This command can be run with
-        # `scout load gene-fusion-report [-r] <case_id> <path/to/research_gene_fusion_report.pdf>`
-        upload_command = ["load", "gene-fusion-report"]
+        upload_command: List[str] = ["load", "gene-fusion-report"]
```

Comment:
You could use your new function to simplify with something like:
```suggestion
        upload_report(case_id=case_id, report_path=report_path, report_type="gene_fusion")
        if research:
            upload_report(case_id=case_id, report_path=report_path, report_type="gene_fusion_research")
```